sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -3104,7 +3108,20 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 						+ "waterType, otherWaterType, infrastructureDetails, otherInfrastructureDetails, waterUse, otherWaterUse, location_id, id FROM tmp_environments");
 				getDao(Environment.class).executeRaw("DROP TABLE tmp_environments");
 
-				// ATTENTION: break should only be done after last version
+            case 348:
+                currentVersion = 348;
+                getDao(EnvironmentSample.class).executeRaw(
+                        "CREATE TABLE environmentsamples(id integer primary key autoincrement, uuid VARCHAR(36) NOT NULL, changeDate TIMESTAMP NOT NULL, "
```

Comment:
I believe the table is supposed to be written in camel case as well, i.e. `enviornmentSamples` - otherwise we could run into issues